server/remote/schedule.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package remote
+
+// ScheduleInformation undocumented
+type ScheduleInformation struct {
+	// Email of user
+	ScheduleID string `json:"scheduleId,omitempty"`
+
+	// 0= free, 1= tentative, 2= busy, 3= out of office, 4= working elsewhere.
```

Comment:
This comment seems to be inaccurate.

Reply:
The availabilities are documented [here](https://docs.microsoft.com/en-us/graph/outlook-get-free-busy-schedule) about halfway down the page